src/Select.jsx

```diff
@@ -102,6 +103,16 @@ export default class Select extends React.Component {
     this.adjustOpenState();
   }
 
+  componentDidMount = () => {
+    if (this.props.autoFocus) {
+      setTimeout(() => {
+        this.setState({ open: true });
```

Comment:
I don't think `focus == open`.

Reply:
That is true, what about opening if there are `<Option>` children?